### ISSUE TITLE
Interface is not fully bilingual

### ISSUE THREAD

**Issue by olafveerman** [orma/openroads-vn-analytics#532]
Reported by Anh

![image](https://user-images.githubusercontent.com/751330/49649654-ca3c7c00-f9f7-11e8-8e8f-9423eb879855.png)

![image](https://user-images.githubusercontent.com/751330/49649659-d0325d00-f9f7-11e8-99fe-80bee2f6665f.png)

**Comment by geohacker** [comment]
Some translations are incomplete. We're waiting on Duc. Duplicate of #495
